lib/apiservers/service/swagger.json

```diff
@@ -527,6 +527,9 @@
         "name": {
           "type": "string"
         },
+        "debug": {
```

Comment:
Consider re-writing the swagger specs in yaml.

Reply:
Filed #6467 so we can address this when we don't have pending PRs modifying the file.